pom.xml

```diff
@@ -2271,11 +2271,15 @@
                     <groupId>org.gaul</groupId>
                     <artifactId>modernizer-maven-plugin</artifactId>
                     <configuration>
+                        <javaVersion>11</javaVersion>
                         <violationsFiles>
                             <violationsFile>${air.main.basedir}/src/modernizer/violations.xml</violationsFile>
                         </violationsFiles>
                         <exclusionPatterns>
                             <exclusionPattern>org/joda/time/.*</exclusionPattern>
+                            <exclusionPattern>com/google/common/collect/.*</exclusionPattern>
```

Comment:
Why these exclusions? If there are new issues here, we should fix those in advance of moving to Java 11, not suppress them.

Reply:
Let me try to remove this.